core/src/main/resources/hudson/PluginManager/sidepanel.jelly

```diff
@@ -0,0 +1,109 @@
+<!--
+The MIT License
+
+Copyright (c) 2022, Jenkins contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<!--
+  Sidepanel component for Plugins Manager
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+  <st:documentation>
+    <st:attribute name="page">
+      Adds the highlighted class to the current page item in the list
+    </st:attribute>
+  </st:documentation>
+
+  <div id="tasks" class="app-plugin-manager__sidebar">
```

Comment:
Why is this still a custom sidepanel implementation?

Reply:
I could do that, (and I have implemented it locally)

but we would lose support for the update count badge.

![image](https://user-images.githubusercontent.com/21194782/185342916-7fdab776-8ed8-4bbf-a7d3-117da464388b.png)

![image](https://user-images.githubusercontent.com/21194782/185343170-c166c6c7-58cc-4388-8a81-57ade8976d3a.png)

any thoughts?

Reply:
As usual, if this is going to be a thing we do, it should be properly supported, e.g. by being added to `l:task`.

Reply:
@janfaracik what do you think shall we add to task now or temporarily remove and enhance later?

Reply:
I've removed it for now